### PR TITLE
[fr] remote-rule-filters

### PR DESCRIPTION
**Pull request by LucieSteib**
Writing antipatterns for AI_FR_HYDRA_LEO_MISSING_COMMA to fix:
-  [[fr] Non necessary comma#5042](https://github.com/languagetooler-gmbh/languagetool-premium/issues/5042)
- [[fr] Comma#5057](https://github.com/languagetooler-gmbh/languagetool-premium/issues/5057)

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml [hunk 1]
```diff
@@ -318,6 +318,26 @@ To ignore a remote rule match, set the <marker> so that it exactly covers the te
...
+                </pattern>
+                <example correction="">La locution <marker>interjective</marker> bon sang, mais c'est bien sûr ! est utilisée.</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <marker>
+                    <token regexp="yes">écriture|orthographe|accord|graphie</token>
+                    </marker>
+                    <token regexp="yes">du|de</token>
+                    <token min="0" max="1" regexp="yes">l'</token>
```
Comment (jaumeortola) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml:336]: "regexp" is not needed here.

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml [hunk 2]
```diff
@@ -318,6 +318,26 @@ To ignore a remote rule match, set the <marker> so that it exactly covers the te
                 </pattern>
                 <example correction="">Par son absence volontaire de <marker>conjonction</marker> de coordination.</example>
             </rule>
+            <rule>
+                <pattern>
+                    <token regexp="yes">locutions?</token>
+                <marker>
+                    <token regexp="yes">interjective?|adverbiale?</token>
+                </marker>
```
Comment (jaumeortola) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml:326]: the plural forms with `s?`
or with inflected="yes"

Comment (jaumeortola) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml:326]: Other possible words (source: https://fr.wiktionary.org/wiki/locution): 
    locution adjective ou locution adjectivale
    locution adverbiale
    locution conjonctive
    locution déterminative
    locution interjective
    locution nominale
    locution postpositive
    locution prépositive ou locution prépositionnelle
    locution pronominale
    locution verbale
    locution-phrase

Comment (LucieSteib) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml:326]: brilliant!

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml (hunk 1 above)
Comment (LucieSteib) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml]: ah, thanks!

**Pull request merged**